### DISCUSSION TITLE
-nW still showing invalid dead subdomains

### DISCUSSION THREAD

**Discussion by Anthonymcqueen21** [projectdiscovery/subfinder#733] [Q&A] [answered]
Hello everyone my ongoing problem is Subfinder not showing valid domains even when I use -nW option it takes a little longer but still showcases dead unused domains. Is there a reason for this any help would be appreciated i am also using the open multiple URLs as well in my hunting it helps but when I attempt to open multiple URLs some load while others don't in this case.

**Comment by olearycrew** [comment, marked as answer]
Hello! Subfinder is only a passive DNS tool, and thus uses the data sources you specify, and thus may return active domains that are marked as active in those sources but aren't actually.

Most of the time, you'll want to pass those results through DNSx or HTTPx to confirm if they are still valid records and responding to requests. Learn more about those projects here: https://nux.gg/dnsx & https://nux.gg/httpx
